test/quantization/fx/test_quantize_fx.py

```diff
@@ -4735,3 +4760,7 @@ def test_resnet18_ddp(self):
         model = models.__dict__[name](pretrained=True).eval().float()
         self._test_model_impl(
             'ddp', 'resnet18', model, eager_quantizable_model)
+
+if __name__ == "__main__":
+    a = TestQuantizeFxModels()
+    a.test_static_gpu_convert_mobilenet()
```

Comment:
can you change this to https://github.com/pytorch/pytorch/blob/master/test/quantization/eager/test_quantize_eager_ptq.py#L1255, we'd like to run tests with python test/test_quantization.py TestCalss.test_case so that we don't need to worry about files (one less thing to remember)

Reply:
oh ya that should have been removed.

Reply:
(fixed)